crates/bevy_scene/Cargo.toml

```diff
@@ -9,7 +9,7 @@ license = "MIT OR Apache-2.0"
 keywords = ["bevy"]
 
 [features]
-default = ["serialize"]
+default = ["serialize", "bevy_render"]
```

Comment:
I'm questioning if this should be a default feature. The motivations for this PR seems to suggest it shouldn't be, and should be enabled in `bevy_internal` or `bevy` proper instead.

Reply:
I've made the change as requested.